protos/build/bazel/remote/execution/v2/BUILD

```diff
@@ -1 +1 @@
-go_package()
+go_package(skip_gofmt=True)
```

Comment:
How come?

Reply:
Because gofmt in later Go versions wants to change the comment formatting. The `.pb.go` file in this directory is generated by `protoc`, I'd rather not have changes in Go version suddenly subject this file to reformatting.